deploy/backplane/lpsre/10-lpsre-addon-operator-cluster.ClusterRole.yml

```diff
@@ -14,4 +14,3 @@ rules:
       - watch
       - patch
       - update
```

Comment:
```suggestion
```

Global scope edit permissions are risky.  I'd prefer we use elevation to gate such operations.  The risk with modifying these without some bumps is the SRE will forget they're being managed and will be reconciled eventually.  No modifications of a managed resource should be trivially easy to do.

Reply:
The Addon API gives LPSREs the ability to pause reconciliation (.spec.pause=True), which is needed for some OLM recovery procedures and helped tremendously in debugging issues in past incidents.

Reply:
Could you drop a comment in the YAML indicating the intent of that access then?  I see it's to specific resources as well, do both of these support pause?

Reply:
Is this the kind of thing that would be good to have a backplane managed script for? (pause/unpause an addon)